### PR TITLE
[LO-309] fix: typeset after async load

### PR DESCRIPTION
**Pull request by hermansje**
### Ticket
Link to ticket: https://datacamp.atlassian.net/browse/LO-309

### What has been done
- Use Mathjax v3, using https://mathjax.github.io/MathJax-demos-web/convert-configuration/convert-configuration.html
- Typeset after dynamic render

### CODE REVIEW

File: assets/js/dependencies/async_loader.js [hunk 1]
```diff
@@ -1,84 +1,81 @@
...
+        cache: false,
+        crossDomain: true,
       });
       /*
       Need to rebind everything when the DOM changes
       */
-      $(document).bind("content-changed",function () {
+      $(document).bind("content-changed", function () {
         Binder.bindGlobalClickHandler();
+        MathJax.typeset();
```
Comment (TimSangster) [inline assets/js/dependencies/async_loader.js:26]: Only things that was added in this file?

File: views/layout.ejs [hunk 2]
```diff
@@ -69,23 +69,24 @@
     <script src="<%= protocol %>//cdnjs.cloudflare.com/ajax/libs/clipboard.js/1.6.0/clipboard.min.js"></script>
     <script src="<%= protocol %>//cdnjs.cloudflare.com/ajax/libs/jquery-modal/0.9.1/jquery.modal.min.js"></script>
     <script src = "<%= protocol %>//cdn.jsdelivr.net/simplemde/latest/simplemde.min.js"></script>
-    <script type="text/x-mathjax-config">
-      MathJax.Hub.Config({
-        messageStyle: "none",
-        jax: ["input/TeX","output/CommonHTML"],
```
Comment (TimSangster) [inline views/layout.ejs:75]: How did you translate these to the new settings below?

**Review by TimSangster — APPROVED** [review]
Looks good, did you test/ is there a way for me to test?

File: assets/js/dependencies/async_loader.js (hunk 1 above)
Comment (hermansje) [reply assets/js/dependencies/async_loader.js:26]: Yes

File: views/layout.ejs (hunk 2 above)
Comment (hermansje) [reply views/layout.ejs:75]: Using the converter linked above

Comment (TimSangster) [reply views/layout.ejs:75]: Woops, missed that, thanks

**Pull request merged**